addon/styles/components/date-picker.scss

```diff
@@ -14,9 +73,26 @@
 
 .cs-component-calendar--nav-dropdown {
   flex: 1;
+  padding: 5px;
 
-  > .ember-power-select-trigger {
-    background: transparent;
+  .ember-power-select-trigger {
     border: none;
   }
 }
+
+
+// cs-theme
+.cs-component-date.cs-theme {
+  --cs-date-background-color: white;
+  --cs-date-body-color: #{$cs-theme-light-gray};
+  font-weight: 600;
+
+  .ember-power-select-trigger:hover,
+  .ember-power-select-search-input:hover {
+    background-color: var(--cs-dropdown-background-color);
+  }
+
+  .ember-power-select-option {
+    border-bottom: none;
+  }
+}
```

Comment:
Should we add some documentation somewhere on how to create a theme, and what vars to change?

Reply:
Sounds good. I'll add that to the docs